models/classes/runner/features/TestRunnerFeatureService.php

```diff
@@ -86,12 +86,17 @@ public function unregister($featureId) {
      */
     public function getAll()
     {
-        $option = $this->getOption(self::OPTION_AVAILABLE);
-        foreach ($option as $feature) {
+        $result = [];
+        /** @var TestRunnerFeatureInterface[] $features */
+        $features = $this->getOption(self::OPTION_AVAILABLE);
+        foreach ($features as $id => $feature) {
             if ($feature instanceof ServiceLocatorAwareInterface) {
                 $feature->setServiceLocator($this->getServiceLocator());
             }
+            if ($feature->isActive()) {
```

Comment:
Could u check if the loaded feature is instance of `TestRunnerFeatureInterface`?

Reply:
Any feature must implement this interface, so I think it's not needed.